gen/config/crates.yaml

```diff
@@ -0,0 +1,17 @@
+# For now, crates with tables we can't generate are commented out
```

Comment:
I picked YAML pretty much arbitrarily; I'm most familiar with it (maybe after JSON, which I did some work with and now have a minor personal vendetta against). I agree that TOML is probably a more consistent choice in a Rust ecosystem.

Now that I've had some time to think about it (and you've pointed out the problems with this config), it seems like the better choice is just to group by (UCD tables) / (IDNA tables) / (etc.), and have flags for generating all of the tables for a major subcrate.

There's no real need for generating for _just_ `unic-ucd-category` (other than maybe if it's just been added).

I'll do a full reconsideration of the config and command line options as I make the switch to TOML.